scripts/skills/perks/perk_rf_discovered_talent.nut

```diff
@@ -16,66 +16,53 @@ this.perk_rf_discovered_talent <- ::inherit("scripts/skills/skill", {
 		this.m.IsRefundable = false;
 	}
 
-	function onAdded()
+	// Is called from player.setAttributeLevelUpValues
+	function addStars()
```

Comment:
You never call this function.

I tested this perk ingame briefly and it doesn't work, which is weird because you marked it as working.
![image](https://github.com/Battle-Modders/mod-reforged/assets/2252464/b2f2371d-8d3b-47ec-b8c1-af5ed1d52db0)

Reply:
Did you pull the branch correctly before testing? The function is called from the hook on player.nut.

Reply:
I have overlooked the changes in the player.nut, they were actually present on the branch I compiled and had locally.
I actually didn't know what I was even needing to do ingame to test the perk so I gave up after forcing a level-up on a char with that perk.